conf/messages

```diff
@@ -122,8 +122,10 @@ employment.employmentSummary.cannotAdd.agent = You cannot add your client’s ex
 
 #################       Add another employment messages        ########################################
 
-AddEmployment.error = Select yes if you want to add an employment
-AddEmployment.header = Do you want to add an employment?
+
+AddEmployment.error = Select yes if you want to add an employer
```

Comment:
The property key name is not consistent. Can you please change that to something like: employment.addEmployment.error